frontend/app/dashboard/src/views/dashboard/groups/GroupMembersView.vue

```diff
@@ -13,6 +13,10 @@
                     </button>
 
                     <button v-if="group && hasFull" class="button icon settings gray" @click="modifyGroup" />
+
+                    <button class="button text" @click="duplicateGroup">
+                        <span class="icon copy" />
+                    </button>
```

Comment:
I would add editing / administration features in the members overview. The members overview should only contain features that are related to viewing and editing members. You should move the duplicate method to the category edit screen (preferred) or the group edit screen.

E.g. when editing the category, there is a list of all the groups in that category. You could add an icon "..." at the right side + right click context menu on the group in that view. When you click on ... or right click, you can show a context menu with the new duplicate feature (and maybe also delete and in the future a move method)

Reply:
Eerder zoiets als dit dan? :) 

![image](https://user-images.githubusercontent.com/12672784/122375578-a1e84d80-cf63-11eb-82ff-f0a54a9a7d0a.png)

Reply:
Yes, perfect! 😍  Maybe but it on both places to make it consequent and easy to find (or am I being too greedy here ^^)

Reply:
so for clarity: both on the general overview of a category as well as when editing the category

Reply:
Hahah not greedy at all! Putting it in both places would be better for consistency! 😋